android/build.gradle

```diff
@@ -65,3 +65,8 @@ task sourcesJar(type: Jar) {
 }
 
 artifacts.add('archives', sourcesJar)
+
+dependencies {
+    testImplementation 'junit:junit:4.12'
+    testCompile "org.mockito:mockito-core:2.+"
```

Comment:
This should be `testImplementation`, too. And can you please specify a concrete version? Otherwise we can't cache this properly.